src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCaseAboutToStartHandlerTest.java

```diff
@@ -16,17 +18,22 @@
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
 import org.mockito.Mock;
 import uk.gov.hmcts.reform.sscs.ccd.callback.Callback;
 import uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType;
 import uk.gov.hmcts.reform.sscs.ccd.domain.*;
+import uk.gov.hmcts.reform.sscs.util.DynamicListLangauageUtil;
 
 @RunWith(JUnitParamsRunner.class)
 public class AdjournCaseAboutToStartHandlerTest {
 
     private static final String USER_AUTHORISATION = "Bearer token";
     private AdjournCaseAboutToStartHandler handler;
```

Comment:
Use inject mocks instead:
```suggestion
    @InjectMocks
    private AdjournCaseAboutToStartHandler handler;
```